fili-core/src/main/java/com/yahoo/bard/webservice/web/TablesApiRequest.java

```diff
@@ -340,6 +380,68 @@ protected LogicalTable generateTable(
         }
     }
 
+    // CHECKSTYLE:OFF
+    public TablesApiRequest withFormat(ResponseFormatType format) {
+        return new TablesApiRequest(format, paginationParameters, uriInfo, builder,
+            tables, table, granularity, dimensions, metrics, intervals, filters);
```

Comment:
Since we're turning checkstyle off anyway, we might as well move all of this onto one line to line up with how we do the other withers.